extension/arquillian-drone-browserstack-extension/src/main/java/org/arquillian/drone/browserstack/extension/local/PlatformUtils.java

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
it seems to me you are reinventing a wheel here, this all could be thrown away, use something like this, it is from apache commons lang, it know almost everything out of the box

https://github.com/arquillian/arquillian-cube/commit/195d36daf884d02eabbf3963b35f6deb1c9f210b